server/src/main/java/org/elasticsearch/action/TaskOperationFailure.java

```diff
@@ -116,7 +111,7 @@ public Exception getCause() {
 
     @Override
     public String toString() {
```

Comment:
I am not sure this method is even needed but I did not want to eliminate it right away.

Reply:
I think it simplifies the debugging, I don't think it is really used for anything else.

Reply:
Thank you. In that case I'll leave it as it is now.